packages/resolver/lib/sources/fs.ts

```diff
@@ -65,18 +65,22 @@ export class FS implements ResolverSource {
     ];
 
     let body, filePath;
-    possiblePaths.forEach(possiblePath => {
+    for (let i = 0; i < possiblePaths.length; i++) {
       try {
+        const possiblePath = possiblePaths[i];
         const resolvedSource = fs.readFileSync(possiblePath, {
           encoding: "utf8"
         });
         body = resolvedSource;
         filePath = possiblePath;
+        
+        return { body, filePath };
       } catch (error) {
         // do nothing
       }
     });
```

Comment:
Looks like the tests are failing cause of this syntax error

```suggestion
    }
```